### ISSUE TITLE
add keyboard shortcut to show type information

### ISSUE THREAD

**Issue by maiermic** [TypeStrong/atom-typescript#957]
I'd like to show type information (at cursor position) by using a keyboard shortcut instead of using the mouse (hover).

**Comment by clarencecastillo** [comment]
Yeah I'm looking for something like this as well. Not sure why, but seems like it was removed from the later releases. See #970

**Comment by lierdakil** [comment]
This was removed during the course of a major rewrite, at one point. Probably by accident, or due to a lack of time, or both. I'll try to take a look and hopefully sneak it in in the next release.

**Comment by clarencecastillo** [comment]
Awesome, thanks!

**Comment by andb0t** [comment]
I see this is closed, but the problem isn't fixed. There are some related issues also closed:
* https://github.com/TypeStrong/atom-typescript/issues/726
https://github.com/TypeStrong/atom-typescript/issues/957 (this one)
* https://github.com/TypeStrong/atom-typescript/issues/1267

This seems to have been a feature in a previous release and somehow was lost during a rewrite. Is there something planned? Currently, the type and linter info fight for space on the screen, which one could solve by having the option for assigning a keyboard shortcut to type info instead of hovering.
